src/solver.f90

```diff
@@ -283,11 +283,11 @@ subroutine divergence_v2p(self, div_u, u, v, w)
       ! Interpolation for v field in x
       ! Interpolation for w field in x
       call self%backend%tds_solve(du_x, u, self%xdirps, &
-                                  self%xdirps%stagder_v2p)
```

Comment:
Also here, I would expect the continuation from the parentheses, isn't this more common?

Reply:
I believe we can play with this parameter
```
       --align_paren[=<n>]
              align continuation lines not preceded
                by '&' with preceding unmatched left parenthesis.
               n=0, or flag omitted: do not align.
               n=1: do align (default).

```
though the default should be do leave it as is